.storybook/main.js

```diff
@@ -1,26 +1,22 @@
-/**
- * For info about getting PostCSS to work with Storybook:
- * https://storybook.js.org/addons/@storybook/addon-postcss
- * https://lifesaver.codes/answer/a-working-example-with-postcss-for-storybook-v5
- * https://blog.jakoblind.no/postcss-webpack/
- */
-
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.@(js|tsx|mdx)'],
+  stories: [
+    '../src/**/*.stories.mdx',
+    '../src/**/*.stories.@(js|jsx|ts|tsx)'
+  ],
+  typescript: {
+    check: false,
+    checkOptions: {},
+    reactDocgen: false,
```

Comment:
^^^ this set to true caused an error when trying to run storybook, I should probably make a comment on that. Also probably don't need the options below.